frontend/components/Domain/Recipe/RecipePage/RecipePage.vue

```diff
@@ -246,12 +254,29 @@ const hasLinkedIngredients = computed(() => {
 
 type BooleanString = "true" | "false" | "";
 
-const edit = useRouteQuery<BooleanString>("edit", "");
+const paramsEdit = useRouteQuery<BooleanString>("edit", "");
+const paramsParse = useRouteQuery<BooleanString>("parse", "");
 
 onMounted(() => {
-  if (edit.value === "true") {
+  if (paramsEdit.value === "true") {
     setMode(PageMode.EDIT);
   }
+
+  if (paramsParse.value === "true") {
+    toggleIsParsing(true);
+  }
+});
+
+watch(isEditMode, (newVal) => {
+  if (!newVal) {
+    paramsEdit.value = undefined;
+  }
+});
+
+watch(isParsing, () => {
+  if (!isParsing.value) {
+    paramsParse.value = undefined;
+  }
```

Comment:
This should remove the params from the URL if they're not true anymore (e.g. instead of `edit=false` it will just go away)